pkg/multicluster/register/helpers.go

```diff
@@ -70,6 +70,9 @@ type RegistrationOptions struct {
 	// List of cluster roles which will be bound to by the created cluster role bindings
 	// The ClusterRoles upserted from the above list will automatically appended to the list
 	ClusterRoleBindings []client.ObjectKey
+
+	// Set of labels to include on the KubernetesCluster resource.
+	KubernetesClusterLabels map[string]string
```

Comment:
wondering if this shouldn't apply to all output resources? and then `ResourceLabels`?

Reply:
makes sense